### ISSUE TITLE
Show Onboarding Tours once and remove "Do not show me this again" checkbox

### ISSUE THREAD

**Issue by alonsogarciapablo** [CartoDB/cartodb#13415]
** **To be done after https://github.com/CartoDB/cartodb/issues/13281 is merged.** **

The goal of this issue is to re-do #13003.

It was already done in #13039 but it also applied to analysis result screen, which should always be displayed. After this change is reverted (https://github.com/CartoDB/cartodb/issues/13281), let's re-implement this (only considering onboarding tour screens).

- [ ] Show Onboarding Tours only once.
- [ ] Remove "Do not show me this again" checkbox for onboarding tour screens
- [ ] Ensure other "onboarding" modals continue being displayed:
  - [ ] Analysis result screens
  - [ ] ?
 
cc: @matallo

**Comment by rubenmoya** [comment]
Hi! I just wanted to know what the correct behavior should be  here, I talked with @arianaescobar and ended we came up with:

- Show general (Builder) onboarding only once
- Show `data`, `style` and `analysis` tabs onboardings only once
- For every analysis applied, show the respective onboarding only once

One of the things we talked about analysis onboardings was showing them only once *per map* instead of *per user*, this will lead us to have:

- General onboarding: ONCE PER USER
- Layer content onboarding: ONCE PER USER
- Analysis onboarding: ONCE PER MAP

What do you think @alonsogarciapablo @urbanphes?

**Comment by alonsogarciapablo** [comment]
> Analysis onboarding: ONCE PER MAP

@ramiroaznar what do you think? do you think this is enough or should it be displayed every single time?

**Comment by ramiroaznar** [comment]
I really like the idea of having an analysis onboarding message once per map. So yes, I think it is enough.

**Comment by alonsogarciapablo** [comment]
👍

**Comment by ramiroaznar** [comment]
Looks good to me!

**Comment by rubenmoya** [comment]
In production! 🚀
